pkg/util/strings.go

```diff
@@ -32,6 +32,14 @@ func SplitString(str string) []string {
 	if len(str) == 0 {
 		return []string{}
 	}
+	if strings.Contains(str, ",") {
+		// Separate by commas
+		substrs := make([]string, 0)
+		for _, s := range strings.Split(str, ",") {
+			substrs = append(substrs, strings.TrimSpace(s))
+		}
+		return substrs
+	}
 
 	return strings.Fields(strings.ReplaceAll(str, ",", " "))
```

Comment:
is this still necessary after parsing per commas in the previous code block?

Reply:
Yep, not necessary